components/ShortEventCardContainer/ShortEventCardContainer.js

```diff
@@ -15,6 +15,7 @@ export default function ShortEventCardContainer({ navigation, events }) {
   }
   let eventCards = events.map(event => <ShortEventCard
     activity={event.activity}
+    key={Math.random()}
```

Comment:
again, possibly a spot we could use the event ID once that is randomly generated